tests/cli/lib/root_account.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+check_root_account() {
+# Try to SSH to a remote machine first using root account using password-based
+# auth (this is expected to fail) and then using key-based auth with the
+# supplied username to check content of /etc/shadow and audit.log.
+#
+# use: check_root_account <user> <machine> [ssh options]
+
+    local ssh_opts="-o StrictHostKeyChecking=no"
+    local user="$1"
+    local machine="$2"
+    ssh_opts="$ssh_opts $3"
+    if [[ "$user" == "" || "$machine" == "" ]]; then
+        rlLogError "check_root_account: Missing user or machine parameter."
+        return 1
```

Comment:
not sure how `return 1` plays in the entire scheme but `rlFail "message"` instead of rlLogError should be better. Need to experiment with it though.

Reply:
You're right that `rlFail` is a better solution, as `rlLogError` just logs the error, but doesn't affect the test result, which might cause silent failures in this case. The `return 1` by itself doesn't affect the test results in any way, it's just meant to skip the further steps from being executed as they would fail anyway.